src/search-params/search-params-manager.js

```diff
@@ -8,7 +8,7 @@ import PropTypes from 'prop-types'
 
 const defaults = {
   apiAccessible: true,
-  sortOrder: 'name_asc',
+  sortOrder: 'start',
```

Comment:
another option might be to have it as a blank string instead of `start` - as in, blank string stands for "unset". If this doesn't show up in the URL either way though, it's probably fine. However, if it is in the URL, I'd be less inclined to accept this as the solution.

Reply:
good point. It doesn't show up in the url.